cultivation/systems/arc_reactor/jarc_reactor/config_schema.py

```
@@ -0,0 +1,194 @@
+    heads: int = 4
+    d_ff: int = 128
+    output_dim: int = 30
+    dropout_rate: float = 0.15
+    context_encoder: ContextEncoderConfig = field(default_factory=ContextEncoderConfig)
+    encoder_dropout_rate: float = 0.61
+    decoder_dropout_rate: float = 0.12
+    lora: LoraConfig = field(default_factory=LoraConfig)
+    checkpoint_path: Optional[str] = None
+
+@dataclass
+class TrainingConfigSchema:
```

Pylint (3.3.7)

[refactor] Too many instance attributes (12/7)

(R0902)